### PR TITLE
Adding dwh

### PR DESCRIPTION
**Pull request by Astarsys**
Added a new Data warehouse API with:
- how-to,
- getting-started
- raw data resources
Added examples for tracker/sensor/batch_list API call
Everything worked well during the tests
Small fix for examples in rules.md - append_zone_title must be inside extended params.

### CODE REVIEW

File: docs/data-warehouse-api/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+The structure of Data Warehouse API is mostly similar to [Backend API](../backend-api/getting-started.md), so if you're familiar with the basics of 
+user API, this will be a great advantage.
+
+***
+
+## Base URL
+
+Data Warehouse API resides in `dwh` subsection of API URL and does not belong to backend APIv2. You need to determine URL 
+to API calls like this:
+*  `https://api.eu.navixy.com/dwh` for European Navixy ServerMate platform.
```
Comment (john9x) [inline docs/data-warehouse-api/getting-started.md:17]: you forget the version - `/dwh/v1`

File: docs/data-warehouse-api/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+***
+
+### Date/time formats
+
+According to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+
+Example: `2023-12-31T23:59:59Z`.
+
+With the parameter iso_datetime=false, date/time output is `yyyy-MM-dd HH:mm:ss`.
```
Comment (john9x) [inline docs/data-warehouse-api/getting-started.md:104]: I propose to add link to https://developers.navixy.com/backend-api/getting-started/#datetime-formats

File: docs/data-warehouse-api/how-to/how-to-request-tracker-raw-data.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+to the platform, and what the data inputs are named. To do this, we need to use `raw_data/get_inputs` request.
+
+Example for a device with ID 123456:
+
+=== "cURL"
+
+    ```shell
+    curl -X 'POST' \
+    'https://api.eu.navixy.com/dwh/v1/tracker/raw_data/get_inputs' \
+    -H 'accept: text/csv' \
```
Comment (john9x) [inline docs/data-warehouse-api/how-to/how-to-request-tracker-raw-data.md:49]: `-H 'accept: text/csv' ` useless

File: docs/data-warehouse-api/getting-started.md (hunk 1 above)
Comment (Astarsys) [reply docs/data-warehouse-api/getting-started.md]: fixed

File: docs/data-warehouse-api/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Data Warehouse API resides in `dwh` subsection of API URL and does not belong to backend APIv2. You need to determine URL 
+to API calls like this:
+*  `https://api.eu.navixy.com/dwh` for European Navixy ServerMate platform.
+*  `https://api.us.navixy.com/dwh` for American Navixy ServerMate platform.
+
+For example, to make raw data readings API request in European Navixy ServerMate, you need to use this URL:
+
+```
+{{ extra.dwh_api_example_url }}/trackers/123/sensors/321/quality
```
[the remark this answers is not in the document]
Comment (Astarsys) [reply docs/data-warehouse-api/getting-started.md]: fixed

File: docs/data-warehouse-api/how-to/how-to-request-tracker-raw-data.md (hunk 3 above)
Comment (Astarsys) [reply docs/data-warehouse-api/how-to/how-to-request-tracker-raw-data.md]: Thank you

**Pull request merged**